### PR TITLE
var-mount: verify udev rules /dev/disk/by-id/scsi-* symlinks present in initramfs

### PR DESCRIPTION
**Pull request by HuijingHei**
Add test script for https://bugzilla.redhat.com/show_bug.cgi?id=1990506

### CODE REVIEW

**Comment by HuijingHei** [comment]
For scsi disk I use `"additionalDisks": ["5G:mpath"]` flag, but it also enables the multipath.

File: tests/kola/var-mount/scsi-id/config.bu [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      wipe_table: true
+      partitions:
+        - number: 1
+          label: var
+  filesystems:
+    - path: /var
+      device: /dev/disk/by-partlabel/var
+      format: xfs
+      wipe_filesystem: true
+      label: var
```
Comment (dustymabe) [inline tests/kola/var-mount/scsi-id/config.bu:15]: can we use [`with_mount_unit`](https://coreos.github.io/butane/examples/#filesystems-and-partitions) here and skip the systemd section below?

File: tests/kola/var-mount/scsi-id/test.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -xeuo pipefail
+# This is to verify udev rules /dev/disk/by-id/scsi-* 
+# symlinks present in initramfs
+# https://bugzilla.redhat.com/show_bug.cgi?id=1990506
+
+# kola: {"platforms": "qemu", "additionalDisks": ["5G:mpath"]}
+
+fstype=$(findmnt -nvr /var -o FSTYPE)
+[[ $fstype == xfs ]]
```
Comment (dustymabe) [inline tests/kola/var-mount/scsi-id/test.sh]: should we add another check of running `lsinitrd /path/to/initramfs` and making sure the files we expect to see are in the output?

File: tests/kola/var-mount/scsi-id/config.bu (hunk 1 above)
Comment (HuijingHei) [reply tests/kola/var-mount/scsi-id/config.bu:15]: Good suggestion, updated

File: tests/kola/var-mount/scsi-id/test.sh (hunk 2 above)
Comment (HuijingHei) [reply tests/kola/var-mount/scsi-id/test.sh]: That makes sense, add check `61-scsi-sg3_id.rules` and `63-scsi-sg3_symlink.rules` in initramfs

**Review by miabbott — APPROVED** [review]
LGTM

**Pull request merged**